src/ui-core/templates/pages/WomensClothing/WomensClothing.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from "react";
+import "./WomensClothing.css";
+import Header from "../../../components/molecules/Header/Header";
+import ItemCard from "../../../components/molecules/ItemCard/ItemCard";
+import FlexContainer from "../../../layouts/FlexContainer/FlexContainer";
+
+export default function WomensClothing() {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    fetch("https://fakestoreapi.com/products/category/women's clothing")
+      .then((res) => res.json())
+      .then((json) => {
+        setItems(json);
+      });
+  }, []);
+
+  return (
+    <>
+      <Header headingText="Modern Walk" />
+      <FlexContainer heading={"Women's Clothing"}>
```

Comment:
@SachinthaSampath Let's rename the `FlexContainer` to `SectionLayout` so that it's more clear what it is!